desc/io/input_reader.py

```diff
@@ -640,7 +640,7 @@ def parse_inputs(self, fname=None):  # noqa: C901 - FIXME: simplify this
         return inputs_list
 
     @staticmethod
-    def write_desc_input(filename, inputs, header=""):
+    def write_desc_input(filename, inputs, header=""):  # noqa: 16 fxn too complex
```

Comment:
syntax for this should be `# noqa: C901` to ignore that specific error, otherwise it ignores everything.